### ISSUE TITLE
First Time Install: Error adding field 

### ISSUE THREAD

**Issue by knightknight** [Mark-H/ClientConfig#92]
- MODX Revolution 2.3.3-pl (advanced)
- ClientConfig 1.3.1-pl

Clean Install of MODx and ClientConfig the only plugin installed.

Error on ClientConfig install:

```
Console running...
Attempting to install package with signature: clientconfig-1.3.1-pl
Package found...now preparing to install.
Grabbing package workspace...
Workspace environment initiated, now installing package...
Error adding field cgSetting->sortorder: Array ( [0] => 42S21 [1] => 1060 [2] => Duplicate column name 'sortorder' ) 
Error adding field cgGroup->sortorder: Array ( [0] => 42S21 [1] => 1060 [2] => Duplicate column name 'sortorder' ) 
Successfully installed package clientconfig-1.3.1-pl
```

**Comment by Mark-H** [comment]
Safe to ignore, not an actual issue.

**Comment by DESIGNfromWITHIN** [comment]
@Mark-H Well might not be a problem, but clients that buy a theme think the install failed somehow...
Any way to hot show these?

**Comment by Mark-H** [comment]
Yeah there's a way to suppress those errors from the build. Will add that in the next release.

**Comment by Mark-H** [comment]
Related there's an actual potential error causing these logs:


PHP notice: Undefined index: Afbeelding met titel
PHP notice: Undefined index: Geordende Lijst

**Comment by Mark-H** [comment]
Fixed in 1.4
